specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/networkSecurityGroup.json

```diff
@@ -659,6 +659,7 @@
             "Udp",
             "Icmp",
             "Esp",
+            "Ah",
             "*"
```

Comment:
In some programming languages, numbers could be assigned to every enum value. In this case, the number would be different. I would suggest to add the new values after the `"*"`.

Reply:
Fixed